assets/js/components/Dashboard/PermanentConfiguration/Workflows/GlEink/ConfigureScreensPage.tsx

```diff
@@ -327,6 +369,7 @@ const ConfigureScreenRow: ComponentType<ConfigureScreenRowProps> = ({
           }}
           placeholder="EIG-"
         />
+        {screenIdError()}
```

Comment:
This column looks like it needs a little more width so the error text doesn't wrap:
![Screenshot 2024-03-04 at 9 46 15 AM](https://github.com/mbta/screenplay/assets/10713153/ccfe81a1-1a71-44aa-bc5e-56472a4ba4c3)